include/s3select.h

```diff
@@ -399,7 +400,7 @@ struct s3select : public bsc::grammar<s3select>
       }
     }
 
-    if (aggr_flow == true)
+    if (aggr_flow)
```

Comment:
== true/false is style issue

Reply:
'== true' is a bit over the line dividing 'style' and 'irregular and strange'...

And, BTW - the 'if' statement semantics is 'if the expression in parens evaluates to 'true' then do ...'.
No new semantic meaning is conveyed by the '== true'. It just adds to the reader's burden.

Reply:
indeed.
there is nothing logical or practical about /x == true/
i probably wanted to remind myself its a boolean and not some number
will change that.

this style appears in our code-base.
[ find ./src  -name '*.cc' -o -name '*.h' | xargs egrep '== false|== true' | grep -v s3select | wc ] ;